### DISCUSSION TITLE
Custom Recipes for Player

### DISCUSSION THREAD

**Discussion by myavuzokumus** [Zrips/CMI#7148] [Feature Requests]
Can add a menu that we can show players for custom recipes only?

In /customrecipes, can show but also players can edit.
In /recipe, showing general recipe. Not for specific custom recipe.

Or if the custom recipe have an ID, it would be nice to open them with the /recipe ID as well.
Sample:
/recipe 10500

**Comment by mrfloris** [comment]
/customrecipes is a staff command, not a player command, the player version is /recipe
